bencher.krun

```diff
@@ -0,0 +1,29 @@
+execfile("luajit.krun", globals()) # inherit settings from the main experiment
+
+MAIL_TO = ["fsfod11@gmail.com"]
+
+PRE_EXECUTION_CMDS = [
+    "sudo systemctl stop apt-daily.timer",
+    "sudo systemctl --runtime disable apt-daily.timer", 
+    "sudo systemctl stop apt-daily-upgrade.timer",
+    "sudo systemctl --runtime disable apt-daily-upgrade.timer",
+    
+    "sudo systemctl stop systemd-timesyncd",
+    "sudo systemctl --runtime disable systemd-timesyncd",
+    "sudo systemctl stop cron",
```

Comment:
Also `atd`

Reply:
krun will fail because systemd will write to stderr that the service doesn't exist.

Reply:
Did you want to take down the interfaces too?

Reply:
Oh right, maybe `atd` has been absorbed in to systemd, as seems to be the trend thesedays.

Reply:
Maybe when its not on bencher3 i mean does krun even run the post commands if some pre commands failed. It just feels brittle with just a list of commands

Reply:
Yes, it always trys to run the post commands. We usually append `|| true` onto the non-essential POST commands, thus allowing them to fail.